test/superlogger_test.rb

```diff
@@ -64,6 +64,41 @@ def output
     assert fields.key?("request_id")
   end
 
+  # The extra fields to log are defined in `/test/dummy/config/application.rb`.
+  # Unfortunately, because `Superlogger::SuperloggerMiddleware` is initialised
+  # together with the Rails application, and because we cannot programmatically
+  # reinitialise the Rails application or run multiple instances of it, we are
+  # only able to test a single Superlogger configuration. Once the Rails
+  # application is initialised, the middleware stack cannot be changed.
```

Comment:
Suggestion, to simplify slightly / improve readability.

```suggestion
  # reinitialise the Rails application or run multiple instances of it, we can
  # only test a single Superlogger configuration. Once the Rails application is
  # initialised, the middleware stack cannot be changed.
```